scripts/postdeploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+cd app
+
+# Run Django migrations.
+python manage.py migrate
+# Seed database with dummy users
+python manage.py seedusers
```

Comment:
it's a guess but I think there is a `pr-postdeploy` script hook that will only run this for review apps